lib/pure/asynchttpserver.nim

```diff
@@ -300,9 +300,10 @@ proc processRequest(
           break
 
         # Read bytesToRead and add to body
-        # Note we add +2 because the line must be terminated by \r\n
-        let chunk = await client.recv(bytesToRead + 2)
-        request.body = request.body & chunk
+        let chunk = await client.recv(bytesToRead)
+        request.body.add(chunk)
```

Comment:
if that's a breaking change we need `-d:nimLegacy<insert good name here>`, as done in other similar cases (with changelog entry + doc comment)

Reply:
It's not a breaking change, the PR that added this code was merged a few days ago.